accepted/2021/default_string_globalization.md

```diff
@@ -0,0 +1,390 @@
+     public new bool Equals(object? x, object? y) { throw null; }
+     public abstract bool Equals(string? x, string? y);
+     public static System.StringComparer FromComparison(System.StringComparison comparisonType) { throw null; }
+     public int GetHashCode(object obj) { throw null; }
+     public abstract int GetHashCode(string obj);
+ }
+ public enum StringComparison
+ {
+-    CurrentCulture = 0,
+-    CurrentCultureIgnoreCase = 1,
+-    InvariantCulture = 2,
+-    InvariantCultureIgnoreCase = 3,
+-    Ordinal = 4,
+-    OrdinalIgnoreCase = 5,
++    Normal = 4, // aliases Ordinal
++    IgnoreCase = 5, // aliases OrdinalIgnoreCase
+ }
+```
+
+## A note on `StringComparison` and `StringComparer`
+
+One notable aspect of this proposal is the complete restructuring of the `StringComparison` and `StringComparer` types. It is important to note that all members of these types will be preserved. The proposal here is to hide all existing fields / static properties and to create new "simpler" names _Normal_ and _IgnoreCase_. Both of these additions alias their ordinal siblings.
+
+> Note: the following paragraphs are speculative. They're based on reasonable lines of thought, but we have not performed the work to draw a conclusive link.
+
+While searching through internal and public sources, we discovered significant usage of the members `StringComparison.InvariantCulture` and `StringComparison.InvariantCultureIgnoreCase`. The usage numbers for these APIs is much higher than would be reasonably expected given their scenario of performing linguistic (somewhat "fuzzy") text matching. This may be explained as an interesting byproduct of having drilled [__CA1305__](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1305) (specify `CultureInfo.InvariantCulture` when calling `ToString(IFormatProvider, ...)`) and ["prefer `string.ToUpperInvariant` over `string.ToUpper`"](https://docs.microsoft.com/dotnet/api/system.string.toupper) into developers' muscle memory over the past decades. This can form a positive association that makes developers believe that __"invariant"__ is always an appropriate default to use.
+
+So when rules like [__CA1307__](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1307) (pass an explicit `StringComparison` value) are enforced, the developer is now forced to choose between ordinal, invariant, or current-culture. It is not reasonable to expect the average developer to understand the difference between ordinal and linguistic behavior. But it _is_ reasonable for an average developer to think to themselves, "I was told _invariant_ was correct in all other situations where I encountered it. It must also be the appropriate default here.". (It would further be reasonable that this decision may be made via muscle memory, with the developer never consciously realizing that they were presented a choice.)
```

Comment:
I don't see the argument for hiding CurrentCulture/CurrentCultureIgnoreCase.  If someone explicitly types "CurrentCulture", that's what they get; it's very clear.